scripts/tasks/download.js

```diff
@@ -18,24 +18,23 @@ var fs = require('fs')
  */
 
 function checkOrFetchFile (file) {
-  let name = file.name
-  let downloadUrl = file.download_url
+  const filePath = path.join(basePath, file.name)
 
-  let localPath = path.join(__dirname, '..', '..', 'locale', 'en', 'blog', 'weekly-updates', name)
-  if (fs.existsSync(localPath)) {
-    console.log(`Weekly Update ${name} exists. (No SHA check, yet.)`)
-    return
-  }
+  fs.access(filePath, function (err) {
+    if (!err) {
+      console.log(`Weekly Update ${filePath} exists. (No SHA check, yet.)`)
+      return
+    }
 
-  console.log(`Weekly Update ${name} does not exist. Downloading.`)
+    console.log(`Weekly Update ${filePath} does not exist. Downloading.`)
 
-  var outputFile = fs.createWriteStream(localPath)
-  https.get(downloadUrl, function (response) {
-    response.pipe(outputFile)
+    https.get(file.download_url, function (response) {
+      response.pipe(fs.createWriteStream(filePath))
```

Comment:
It would help to understand what file the message refers to, otherwise with multiple files, it can be confusing but ye a simple "Download completed" can work too.